src/components/textbox/textbox.pw.tsx

```diff
@@ -709,7 +709,7 @@ test.describe("Prop checks for Textbox component", () => {
     mount,
```

Comment:
**nitpick(non-blocking):** It won't let me comment on the test description here. It may be worth changing the wording in this test description from `inputRef` for just `ref` just to avoid confusion.